modules/web/src/main/scala/observe/ui/components/Home.scala

```diff
@@ -85,81 +86,94 @@ object Home {
           tabOperations = TabOperations.Default
         ).some
       )
-      .render { (props, _, observations, demo) =>
-        <.div(ObserveStyles.MainUI)(
-          Divider(position = Divider.Position.HorizontalCenter, clazz = ObserveStyles.Divider)(
-            "Observe GS"
-          ),
-          Splitter(
-            layout = Layout.Vertical,
-            stateKey = "main-splitter",
-            stateStorage = StateStorage.Local,
-            clazz = ObserveStyles.Shrinkable
-          )(
-            SplitterPanel()(
-              Splitter(
-                stateKey = "top-splitter",
-                stateStorage = StateStorage.Local,
-                clazz = ObserveStyles.TopPanel
-              )(
-                SplitterPanel(size = 80)(
-                  observations.toPot.render(SessionQueue(_))
-                ),
-                SplitterPanel()(
-                  HeadersSideBar(
-                    props.rootModel.get.status,
-                    props.rootModel.get.operator,
-                    props.rootModel.zoom(RootModel.conditions)
-                  )
-                )
-              )
+      .render: (props, ctx, observations, demo) =>
+        import ctx.given
+
+        props.rootModel.get.userVault.map(userVault =>
```

Comment:
Are we going to have side tabs like in explore?

Reply:
I hadn't thought of that. Do you think it would be useful?